engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -218,4 +220,11 @@
      */
     // TODO: transform this to return an object or a map. Consumers would then take care of formatting.
     String getMetrics();
+
+    /**
+     * Returns the RenderGraph used to organize the nodes. This can be used to Modules to add/remove Nodes to the game.
```

Comment:
This needs more work, I'm aware of that.

Reply:
```
* Returns the RenderGraph.
*
* This object is used by Engine and Modules to add/remove Nodes to/from the rendering process.
*
* @return the RenderGraph containing the nodes used by the rendering process.
```